index.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
+require('array-includes/polyfill');
```

Comment:
Maybe just use this as a library so you don't have to setup jest to polyfill this separately:

```js
import includes from 'array-includes';
```